spk/bazarr/Makefile

```diff
@@ -0,0 +1,57 @@
+SPK_NAME = bazarr
+
+include ../../cross/bazarr/bazarr.mk
+
+SPK_VERS = $(BAZARR_VERSION)
+SPK_REV = 2
+
+SPK_ICON = src/bazarr.png
+
+REQUIRED_DSM = 6.1
+
+BUILD_DEPENDS = cross/python cross/setuptools cross/pip cross/wheel
```

Comment:
I am almost sure this variable is useless - please compare how `cross/sabnzbd` and `spk/sabnzbd` are set up to generate wheels without building whole python stack.

Reply:
I only managed to remove cross/pip from the build process (required for lxml actually)